src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 15px;
+  margin-right: 48px;
```

Comment:
try to avoid external geometry for block elements, you can move to container__card element